index2.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles2.css">
+    <title>Concept pagina</title>
+</head>
+
```

Comment:
Algemeen: CleanCode practices; de code oogt soms wat rommelig, kleine slordigheidjes zoals .classname{ , Engels en Nederlandse term als class (zie regel 35 en 47), hoe de code nu is uitgelijnd, is het soms wat moeilijk om er goed doorheen te lezen. Het commando CTRL + ALT + L helpt hier al een hoop bij.